src/Public/js/nipsweb.managelibrary.js

```diff
@@ -4,8 +4,13 @@ var Library = function () {
             'audio/mp3', 'audio/x-mp3', 'audio/mpg', 'audio/mpg3', 'audio/mpegaudio'];
     var allowed_all = ['audio/mpeg3', 'audio/x-mpeg-3', 'audio/mpeg', 'audio/x-mpeg',
             'audio/mp3', 'audio/x-mp3', 'audio/mpg', 'audio/mpg3', 'audio/mpegaudio', 'audio/wav', 'audio/x-wav',
-            'audio/mp4a-latm', 'audio/mp4', 'audio/aac']
+            'audio/mp4a-latm', 'audio/mp4', 'audio/aac'];
 
+    var icon_error = '<div class="glyphicon glyphicon-exclamation-sign"></div>&nbsp;';
+    var icon_ok = '<div class="glyphicon glyphicon-ok"></div>&nbsp;';
+    var icon_ok = '<div class="glyphicon glyphicon-ok"></div>&nbsp;';
+    var icon_loading = '<div class="glyphicon glyphicon-refresh gly-spin"></div>&nbsp;';
+    var icon_close = '<a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>';
```

Comment:
In ES6 you have const and let instead of var, well var is still there but you're not supposed to use it.

Looks like const has decent browser support.